esmvaltool/diag_scripts/cvdp/cvdp/driver.ncl

```diff
@@ -2,68 +2,68 @@
+  webpage_title     = "Title goes here"          ; Set webpage title
 
-;---Advanced Options----------------------------------------------------------------------                                             
+  tar_output        = "False"               ; True = tar up all output in outdir and remove individual files, False = do not
+                                            ; Note: ALL files in outdir will be tarred up and then removed from the outdir directory.
+
+;---Advanced Options----------------------------------------------------------------------
   zp = "ncl_scripts/"    ; directory path of CVDP NCL scripts. (must end in a "/")
                          ; Examples: "ncl_scripts/" if all code is local, or on CGD or CISL systems: "~asphilli/CESM-diagnostics/CVDP/Release/v4.1.0/ncl_scripts/"
-                         ; Regardless of this setting the following files should be in one directory: namelist, driver.ncl, and namelist_obs. 
+                         ; Regardless of this setting the following files should be in one directory: namelist, driver.ncl, and namelist_obs.
                          ; If pointing to code in ~asphilli make sure the driver script version #s match between this script and the script in ~asphilli.
-  
-  ncl_exec = "ncl"       ; This can be changed to a different path if a different version of NCL needs to be used, such as "/different/path/to/bin/ncl"                           
-                         
-  run_style = "parallel"   ; parallel = allow simple python-based parallelization to occur. X number of CVDP NCL scripts will be called at once. 
+
+  ncl_exec = "ncl"       ; This can be changed to a different path if a different version of NCL needs to be used, such as "/different/path/to/bin/ncl"
+
+  run_style = "parallel"   ; parallel = allow simple python-based parallelization to occur. X number of CVDP NCL scripts will be called at once.
                          ;            X is set via max_num_tasks. Terminal output will be harder to follow.
-                         ; serial = call CVDP NCL scripts serially. (Default)    
-                         
-  max_num_tasks = 4      ; if run_parallel = "parallel", how many CVDP NCL scripts can be called at once? (If unsure set to 3)     
-  
-  modular = "False"      ; True = Run only those CVDP scripts specified in modular_list.
+                         ; serial = call CVDP NCL scripts serially. (Default)
+
+  max_num_tasks = 4      ; if run_parallel = "parallel", how many CVDP NCL scripts can be called at once? (If unsure set to 3)
+
+  modular = "True"      ; True = Run only those CVDP scripts specified in modular_list.
                          ; False = Run all CVDP scripts (Default)
 
-  modular_list = "pdo,aice.trends_timeseries,sst.indices"  ; When modular = "True" list the CVDP scripts that will be run. 
+  modular_list = "sst.indices,psl.nam_nao"  ; When modular = "True" list the CVDP scripts that will be run.
```

Comment:
Most of the changes made to `esmvaltool/diag_scripts/cvdp/cvdp/driver.ncl` are related to formatting but not only. Could you please say if the modifications above are necessary? Has the CVDP package been updated upstream? Since the package is also used for other work (and the `recipe_cvdp.yml`), we need to make sure that these changes are necessary and backward compatible.